doc/managing-the-glusterd2-service.md

```diff
@@ -0,0 +1,54 @@
+# Managing the glusterd Service
+
+After installing GlusterFS and Glusterd2, you must start `glusterd2`
+service. The glusterd2 service serves as the Gluster elastic volume
+manager, overseeing glusterfs processes, and co-ordinating dynamic
+volume operations, such as adding and removing volumes across multiple
+storage servers non-disruptively.
+
+This section describes how to start the glusterd2 service in the
+following ways:
+
+**Note**: You must start glusterd2 on all GlusterFS servers.
+
+## Gluster user group
+
+On each server nodes, create "gluster" user group if not exists
+already using the following command.
+
+    # groupadd gluster
+
+Users from this group can run `glustercli` commands in GlusterFS
+servers.
+
+**Note**: Create "gluster" group before starting `glusterd2`
+
+## Starting and Stopping glusterd2 Manually
```

Comment:
Do we have to talk about starting from binaries? Need more thoughts here.
Thought of documenting how to install from rpm as well as the binaries. Do we need to have this as well or is it better to put that section as a separate link?

Reply:
I prefer not to mention about running binaries directly. It may lead to different non-standard configuration(No config file or different values in custom config file).

Also glusterd2 binary will not demonize by default, that may become issues.

Reply:
It may make sense to adjust the title to read "Starting and Stopping glusterd2 Service Manually" and then there's less of a reason to mention binaries as systemd will be the most typical way to start and stop services on most of the systems where glusterd2 is installed (standalone).

Anyone running glusterd2 standalone with something that isn't systemd is kind of on their own IMO but they ought to be used to managing services with whatever they have anyway.